packages/perseus/src/types.ts

```diff
@@ -44,6 +49,12 @@ export type SerializedState = Record<string, any>;
  * scoring, state serialization/deserialization, and focus management.
  */
 export interface Widget {
+    /**
+     * don't use isWidget; it's just a dummy property to help TypeScript's weak
+     * typing to recognize non-interactive widgets as Widgets
+     * @deprecated
```

Comment:
Yeah 😞 , maybe we can eventually expand this to have a second interface like `InteractiveWidget` which contains all the functions that truly interactive widgets implement.